crates/teloxide/src/dispatching/handler_ext.rs

```diff
@@ -93,3 +112,38 @@ where
         message.text().and_then(|text| C::parse(text, &bot_name).ok())
     })
 }
+
+/// Returns a handler that accepts a parsed command `C` if the command
+/// contains a bot mention, for example `/start@my_bot`.
+///
+/// A call to this function is the same as
+/// `dptree::entry().filter_mention_command()`.
+///
+/// See [`HandlerExt::filter_mention_command`].
+///
+/// ## Dependency requirements
+///
+///  - [`crate::types::Message`]
+///  - [`crate::types::Me`]
+#[must_use]
+pub fn filter_mention_command<C, Output>(
+) -> Handler<'static, DependencyMap, Output, DpHandlerDescription>
+where
+    C: BotCommands + Send + Sync + 'static,
+    Output: Send + Sync + 'static,
+{
+    dptree::filter_map(move |message: Message, me: Me| {
+        let bot_name = me.user.username.expect("Bots must have a username");
```

Comment:
I think, it's safe to unwrap here, bot always has username

Reply:
filter_command has this, i copied it, i think it's better to be consistent